tools/statsnoop.py

```diff
@@ -139,6 +139,8 @@ def try_attach_syscall_probes(syscall):
 try_attach_syscall_probes("statfs")
 try_attach_syscall_probes("newstat")
 try_attach_syscall_probes("newlstat")
+try_attach_syscall_probes("fstatat64")
```

Comment:
How about this one? should it belong to `yscall__statx_entry`?

Reply:
Yes. It should reuse the `statx_entry`.